src/apigateway/templates/apigTemplates.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const APIG_REMOTE_INVOKE_TEMPLATE = `
```

Comment:
Can we localize the text in this template?

Reply:
We have like 4 or 5 templates like this, I think we should fix localization of these things later

Reply:
On one hand, yes, but also we're going through the motions of messaging in localized error messages.